sql-queries/sql-column-name/sql-column-name-sqlserver.sql

```diff
@@ -0,0 +1,10 @@
+SELECT COLUMN_NAME
```

Comment:
I don't think this is needed for this tutorial, can you remove it from the PR, please?

Reply:
You can remove this from the PR by deleting the file on your filesystem and then running

```
git add .
git push
```